tests/new_unit_tests/bootstrap.php

```diff
@@ -6,3 +6,10 @@
 require_once __DIR__ . '/../../vendor/autoload.php';
 
 \VCR\VCR::configure()->enableRequestMatchers(['method', 'url', 'body',]);
+
+
+// Override the default cache directory by setting an environment variable. This prevents our tests from overwriting
+// the user's real cache and session.
+// @TODO: Unit tests should not rely on side effects like these. When the Config object is properly injectable this
+// will not be necessary.
+putenv("TERMINUS_CACHE_DIR=~/.terminus/testcache");
```

Comment:
Yeah. I thought about abstracting this because hardcoding == bad. But TBH, this is temporary. Unit tests REALLY shouldn't be writing random stuff to disk based on an environment variable. Once I get a chance to pull the self-configuration stuff from Session/FileCache then this won't be needed at all. The 'FileCache' part of the session will be mockable and nothing will get written to disk during the session unit tests.

If there's a practical need to be able to overwrite this in the mean time while people are working on the codebase then I can certainly make this configurable but I don't want to validate a bad practice by giving making it look nice.